src/libraries/System.Private.CoreLib/src/System/Half.cs

```diff
@@ -1475,16 +1488,23 @@ public static Half BitDecrement(Half x)
             // Negative values need to be incremented
             // Positive values need to be decremented
 
-            bits += (ushort)(((short)bits < 0) ? +1 : -1);
-            return new Half(bits);
+            if (IsNegative(x))
+            {
+                bits += 1;
+            }
+            else
+            {
+                bits -= 1;
```

Comment:
Nit: Curious for the reason for prefering this over ++ and -- ?

Reply:
Just my preferred style. I personally don't like `x++` or `x--` as a standalone expression (in loops its fine), as I find it makes the code much harder to read/reason about. The `x += 1` and `x -= 1` instead makes it very clear what's going on and that an assignment/mutation is happening here.